conf.yaml

```diff
@@ -190,6 +189,12 @@ repository:
                 -
                     action: assign_tag
                     action_value: 'popularity/low'
+                 -
+                    action: remove_tag
+                    action_value: 'popularity/medium
```

Comment:
@martenson huh, that first link only works if you've expanded the correct portion. Ugh. Sorry.

It points at this:

```
                 - has_tag__not: 'area/.*'
                 - has_tag__not: 'kind/.*'
                 - has_tag__not: 'status/.*'
                 - has_tag__not: 'merge'
```

I mean that instead of doing `self.issue.remove_from_labels(tag_name)`, do 

```python
for tag in self.issues.get_tags(): # pseudocode 
  if compiled_regex_tag_matcher.match(tag):
    self.issue.remove_from_labels(tag_name)
```